packages/salesforcedx-apex-debugger/test/unit/adapter/apexDebug.test.ts

```diff
@@ -706,6 +723,10 @@ describe('Interactive debugger adapter - unit', () => {
       clock.restore();
     });
 
+    afterAll(() => {
```

Comment:
It looks like the construction of adapter, as well as the deconstruction, should be moved one level up (to the root, if you will).

Reply:
Yeah I can try and clean this up. This test in general is pretty bananas.  It has around 20 describe blocks.  I'll make a note to create a ticket to clean this up eventually.